src/homepageExperience/components/steps/Tokens.tsx

```diff
@@ -62,7 +64,11 @@ export const Tokens: FC<OwnProps> = ({
   )
 
   useEffect(() => {
-    dispatch(getAllResources())
+    try {
+      dispatch(getAllResources())
```

Comment:
Could you include a screenshot testing it both ways - i.e., showing that the dispatch works and notifies the user; and also notifies the user when there's a failure?

To test that it's not working, go into the thunk and `throw Error()` in the location where you'd expect to hear back from the quartz or idpe APIs.